docs/cookbook/taxation/customize-tax-by-address.rst

```diff
@@ -0,0 +1,36 @@
+How to configure tax rates to be based on shipping address?
+===========================================================
+
+The default configuration of Sylius tax calculation is based on billing address but there are situations where we would
+like to use a shipping address to be used in this process. This may be useful to anyone who uses Sylius in European Union
+as from 1st July 2021 the new taxation rules will be applied.
+
+.. note::
+
+    You can learn more about new EU taxation rules `here <https://ec.europa.eu/taxation_customs/business/vat/modernising-vat-cross-border-ecommerce_en>`_.
+
+To change the way how the taxes are calculated: by billing or by shipping address, you need to declare ``OrderTaxesProcessor`` with
+additional argument in your config file:
+
+.. code-block:: yaml
+
+    # app/config/services.yaml
+        App\OrderProcessing\OrderTaxesProcessor:
+        arguments:
+            - '@sylius.provider.channel_based_default_zone_provider'
+            - '@sylius.zone_matcher'
+            - '@sylius.registry.tax_calculation_strategy'
+            - '@sylius.taxation_address_resolver'
+        tags:
+            - { name: sylius.order_processor, priority: 10 }
```

Comment:
If I see correctly, there is no need for this part

Reply:
- '@sylius.taxation_address_resolver'
this param is optional, and it needs to be added in order to use resolver

Reply:
This parameter is optional not to provide bc breaks if someone before Sylius 1.11 overwritten this service, but here in 1.11, this service is configured in Sylius out of the box, so adding this part can be more confusing than helpful imo.